lib/main_common.pm

```diff
@@ -859,11 +859,11 @@ sub load_inst_tests {
             if (defined(get_var("RAIDLEVEL"))) {
                 loadtest "installation/partitioning_raid";
             }
-            elsif (check_var('LVM', 0) && get_var('ENCRYPT')) {
-                loadtest 'installation/partitioning_crypt_no_lvm';
+            elsif (get_var('LVM')) {
+                load_lvm_tests();
             }
-            elsif (get_var("LVM")) {
-                loadtest "installation/partitioning_lvm";
+            elsif (check_var('LVM', 0) && get_var('ENCRYPT')) {
+                loadtest 'installation/partitioning/no_lvm_encrypt';
```

Comment:
I suggest to keep the old name and leave that for a separate PR. IMHO "no_lvm_encrypt" can be misleading as meaning "no_lvm" but "encrypt" or "no_lvm" and also "no_encrypt"

Reply:
Namings changed to be more clear.